.semaphore/semaphore.yml

```diff
@@ -20,6 +20,11 @@ promotions:
         branch:
           - ^master$
 
+  - name: Stage PRs for review 
+    pipeline_file: pr-staging-deploy.yml
+    auto_promote:
+      when: "branch != 'master' or branch != 'release' or branch !~ '^dependabot\.'"
```

Comment:
```suggestion
      when: "branch != 'master' and branch != 'release' and branch !~ '^dependabot\.'"
```

Reply:
Thanks,  @colinhicks, that is a better condition